frontend/src/lib/components/common/badge/model.ts

```diff
@@ -0,0 +1,9 @@
+import type { IconProps } from 'svelte-awesome/components/Icon.svelte'
+
+type BaseColor = 'blue' | 'gray' | 'red' | 'green' | 'yellow' | 'indigo' | 'purple' | 'pink'
+export const ColorModifier = 'dark-'
```

Comment:
Dark mode should not be set that way:  
Each color (blue, gray, ...) should define its dark colors using the tailwind `dark:` variant.
[Tailwind Dark mode documentation](https://tailwindcss.com/docs/dark-mode)